helpers/class.ResultTransmitter.php

```diff
@@ -182,6 +185,11 @@ private static function transformValue($value) {
             if ($value instanceof QtiFile) {
                 return taoQtiCommon_helpers_Utils::qtiFileToString($value);
             }
+            if ($value instanceof RecordContainer) {
+                $marshaller = new Marshaller;
+                return $marshaller->marshall($value);
+            }
+
```

Comment:
PSR12: Expected 1 space after closing brace; newline found

Reply:
I'd recommend to check the file to see if there are more issues like this...

Reply:
would you mind to create a ticket for enabling the ci as we have for the extensions on this library?

Reply:
Yes. I tried to include missing GitHub files but it seems not working for this lib.